src/inversify.config.ts

```diff
@@ -11,16 +11,20 @@
  */
 
 import { Container } from 'inversify';
+// import { NavbarAlerts } from './services/alerts/navbarAlerts';
```

Comment:
please remove commented lines here and below in this config.

Reply:
I have done